code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -100,16 +100,6 @@
 	..()
 	reagents.add_reagent(TOXIN, 30)
 
-/obj/item/weapon/reagent_containers/glass/bottle/insecticide
-	name = "insecticide bottle"
-	desc = "A small bottle of insecticide. Do not pour on insectoid colleagues."
-	icon = 'icons/obj/chemical.dmi'
-
-/obj/item/weapon/reagent_containers/glass/bottle/insecticide/New()
-	..()
-	reagents.add_reagent(INSECTICIDE, 30)
-
-
```

Comment:
Don't delete the insecticide bottles, they're used on the NutriMax vendor (the one with the chems and stuff).